ui-core/src/components/EditAddress.tsx

```diff
@@ -86,6 +88,22 @@ export function EditAddress(
     return undefined
   }
 
+  const labels = i18n ? {
+    'street1': i18n('street1'),
+    'street2': i18n('street2'),
+    'city': i18n('city'),
+    'country': i18n('country'),
+    'postalCode': i18n('postalCode'),
+    'state': i18n('state')
+  } : {
+    'street1': 'Street 1',
+    'street2': 'Street 2',
+    'city': 'City',
+    'country': 'Country',
+    'postalCode': 'Postal Code',
+    'state': 'State'
+  }
```

Comment:
this is kind of gnarly -- why do we need these defaults specified here?

Reply:
Again, because it's in core :'( I am not 100% this is the best way to do it, but I am not sure what would be better.